examples/basics/parameters/README.md

```diff
@@ -0,0 +1,15 @@
+# Basics: Parameter passing
+
+This example shows, how to use the different ways to pass parameters. You can pass them by name or by position
```

Comment:
Can this sentence be enhanced? E.g. "This examples shows how parameters can be passed either by name or by position"